### ISSUE TITLE
Style/YodaCondition: Variables accessed with safe navigation operator behave as if they were literals

### ISSUE THREAD

**Issue by backspac** [bbatsov/rubocop#4413]
## Expected behavior

No offenses are detected

## Actual behavior

```
test.rb:1:10: C: Reverse the order of the operands a&.b == c.
p 'a' if a&.b == c
         ^^^^^^^^^
```

## Steps to reproduce the problem

```shell
$ echo "p 'a' if a&.b == c" > test.rb
$ rubocop test.rb
```

## RuboCop version

```
$ rubocop -V
0.49.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin15)
```

**Comment by bbatsov** [comment]
That's already fixed on `master`.
